packages/jest-message-util/src/index.js

```diff
@@ -221,7 +224,14 @@ const formatPaths = (config: StackTraceConfig, relativeTestPath, line) => {
   return STACK_TRACE_COLOR(match[1]) + filePath + STACK_TRACE_COLOR(match[3]);
 };
 
-const getTopFrame = (lines: string[]) => {
+export const getStackTraceLines = (
+  stack: string,
+  options: StackTraceOptions = {noStackTrace: false},
+) => {
+  return removeInternalStackEntries(stack.split(/\n/), options);
```

Comment:
can you get rid of the braces and return? :P

Reply:
Could add https://eslint.org/docs/rules/arrow-body-style (which is autofixable)

Reply:
#6406